src/module.c

```diff
@@ -439,17 +439,17 @@ typedef struct RedisModuleKeyOptCtx {
 } RedisModuleKeyOptCtx;
 
 /* Data structures related to redis module configurations */
-/* The function signatures for module config get callbacks. These are identical to the ones exposed in redismodule.h. */
+/* The function signatures for module config get callbacks. These are identical to the ones exposed in valkeymodule.h. */
 typedef RedisModuleString * (*RedisModuleConfigGetStringFunc)(const char *name, void *privdata);
 typedef long long (*RedisModuleConfigGetNumericFunc)(const char *name, void *privdata);
 typedef int (*RedisModuleConfigGetBoolFunc)(const char *name, void *privdata);
 typedef int (*RedisModuleConfigGetEnumFunc)(const char *name, void *privdata);
-/* The function signatures for module config set callbacks. These are identical to the ones exposed in redismodule.h. */
+/* The function signatures for module config set callbacks. These are identical to the ones exposed in valkeymodule.h. */
```

Comment:
Please don't change this just yet.

```suggestion
/* The function signatures for module config set callbacks. These are identical to the ones exposed in redismodule.h. */
```